packages/web3-shared/evm/pipes/index.ts

```diff
@@ -232,6 +236,8 @@ export function resolveCollectibleLink(
             return ''
         case NonFungibleAssetProvider.NFTSCAN:
             return ''
+        case NonFungibleAssetProvider.TREASURE:
+            return ''
```

Comment:
Can we compose a link like this: `https://marketplace.treasure.lol/collection/legion-genesis/2036`?